pkgs/os-specific/linux/tuxedo-control-center/default.nix

```diff
@@ -0,0 +1,159 @@
+{ pkgs, lib, stdenv, makeDesktopItem, desktop-file-utils,
+
+  python,
+
+  makeWrapper, nodejs, electron_9, fetchFromGitHub
+}:
+
+let
+  baseName = "tuxedo-control-center";
+  version = "1.0.14";
+
+  baseNodePackages = (import ./node-composition.nix {
+    inherit pkgs nodejs;
+    inherit (stdenv.hostPlatform) system;
+  });
+
+  nodePackages = baseNodePackages.package.override {
+    src = fetchFromGitHub {
+      owner = "tuxedocomputers";
+      repo = "tuxedo-control-center";
+      rev = "v${version}";
+      sha256 = "00gpsvjbli9f6vj31lbs7flmcz215r91iz6rg1kbzklc3y5k2ifj";
+    };
+
+    # Electron tries to download itself if this isn't set. We don't
+    # like that in nix so let's prevent it.
+    #
+    # This means we have to provide our own electron binaries when
+    # wrapping this program.
+    ELECTRON_SKIP_BINARY_DOWNLOAD=1;
+
+    # Angular prompts for analytics, which in turn fails the build.
+    #
+    # We can disable analytics using false or empty string
+    # (See https://github.com/angular/angular-cli/blob/1a39c5202a6fe159f8d7db85a1c186176240e437/packages/angular/cli/models/analytics.ts#L506)
+    NG_CLI_ANALYTICS="false";
+  };
+
+  desktopItem = makeDesktopItem {
+    name = "tuxedo-control-center";
+    exec = "tuxedo-control-center %U";
+    comment = "An application helping you to tune your TUXEDO";
+    desktopName = "TUXEDO Control Center";
+    icon = "tuxedo-control-center";
+    categories = "System;";
+  };
+in
+
+stdenv.mkDerivation rec {
+  name = "${baseName}-${version}";
+  src = "${nodePackages}/lib/node_modules/tuxedo-control-center/";
+
+  nativeBuildInputs = [
+    desktop-file-utils # for desktop-file-validate
+  ];
+
+  buildInputs = [
+    nodejs
+    makeWrapper
+
+    # For node-gyp
+    python
```

Comment:
```suggestion
    python2
```

Reply:
I've changed this to use python3, which works fine as well. No reason to add more dependencies on python2.